api/application/client_test.go

```diff
@@ -1182,6 +1182,41 @@ func (s *applicationSuite) TestScaleApplicationArity(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, "expected 1 result, got 2")
 }
 
+func (s *applicationSuite) TestScaleApplicationValidation(c *gc.C) {
```

Comment:
Not a validation test, but we're missing a test for success where scale change is used but not scale. ie a test similar to TestScaleApplication() but for scale change